### PR TITLE
Update dotnet sln for 3.0

### PR DESCRIPTION
**Pull request by mairaw**
Contributes to #9893 and #9256

[Internal review url](https://review.docs.microsoft.com/en-us/dotnet/core/tools/dotnet-sln?branch=pr-en-us-15545)

### CODE REVIEW

File: docs/core/tools/dotnet-sln.md [hunk 1]
```diff
@@ -32,61 +31,141 @@ To use the `dotnet sln` command, the solution file must already exist. If you ne
...
+
+#### Arguments
+
+- **`SOLUTION_FILE`**
+
+  The solution file to use. If not specified, the command searches the current directory for one. If there are multiple solution files in the directory, one must be specified.
+
+- **`PROJECT_PATH`**
+
+  The path to the project to remove from the solution. Remove multiple projects by adding one after the other separated by spaces. [Globbing patterns](https://en.wikipedia.org/wiki/Glob_(programming)) are also supported on Unix/Linux-based terminals.
```
Comment (BillWagner) [inline docs/core/tools/dotnet-sln.md]: I'm concerned about the sentence on globbing. My understanding is that the unix shell expands glob patterns before invoking the `dotnet` command. Adding this statement makes me wonder if globbing is implemented in the command. If it was, I would think I should quote the pattern:  `dotnet sln remove '*.*'`. I don't that's what you meant. If globbing is performed by the shell, I'd reword it as follows:

```suggestion
  The path to the project to remove from the solution. Remove multiple projects by adding one after the other separated by spaces. Unix/Linux shell [globbing pattern](https://en.wikipedia.org/wiki/Glob_(programming)) expansions are processed correctly by the `dotnet sln` command.
```

**Review by BillWagner — APPROVED** [review]
Overall this looks great to me.

I had one question that we should resolve with @dsplaisted After that, it's ready to :shipit:

File: docs/core/tools/dotnet-sln.md (hunk 1 above)
Comment (mairaw) [reply docs/core/tools/dotnet-sln.md]: Yes, they're performed by the shell, not by the CLI.
